app/views/foreman_bootdisk/host.erb

```diff
@@ -27,17 +41,16 @@ goto loop_success
 :loop_success
 echo Configuring net${idx} for static IP address <%= interface.ip %>
 ifopen net${idx}
-set netX/ip <%= interface.ip %>
-set netX/netmask <%= interface.subnet.mask %>
-<% if interface.subnet.gateway.present? %>
-set netX/gateway <%= interface.subnet.gateway %>
+set net${idx}/ip <%= ip %>
+set net${idx}/netmask <%= mask %>
+<% if gw.present? %>
+set net${idx}/gateway <%= gw %>
```

Comment:
I noticed we actually render `set netX/ip` there which seems incorrect!? Dunno how this could work.